src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -1076,6 +1142,35 @@ def _get_latest_revision(bucket_paths):
   return None
 
 
+def _maybe_get_http_build_url(revision) -> Optional[str]:
+  """Tries to get the HTTP build URL if it exists.
+
+  Args:
+      revision: the revision we're currently handling.
+
+  Returns:
+      the build URL.
+  """
+  http_build_url_pattern = environment.get_value('RELEASE_BUILD_URL_PATTERN')
```

Comment:
RELEASE_BUILD_URL_PATTERN isn't actually used anywhere. I don't think we should use it and we should probably remove it from the job definitions. Instead we should use RELEASE_BUILD_BUCKET_PATH